system-int-test/test/cmr/system_int_test/ingest/provider_ingest_test.clj

```diff
@@ -96,13 +96,37 @@
             "PROV0F" "S0F" false true "Group1"
 
             "- two consortiums"
-            "PROV10" "S10" true true "Group2,Group3"
+            "PROV10" "S10" true true "Group2 Group3"
 
-            "- two consortiums with a space"
-            "PROV11" "S11" nil nil "Group2, Group3"
+            "- two consortiums with an underscore"
+            "PROV11" "S11" true true "Group2_1 Group2_3"
 
             "- all blank"
```

Comment:
I see you treated consortiums with only spaces as valid. I think this is ok. But I would change the parsing to only return one `nil` rather than multiple `nil` consortiums.

Reply:
Done.